python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -228,6 +260,16 @@ def _mg_call_plc_uniform_neighbor_sample(
         else create_empty_df(indices_t, weight_t)
     )
 
+    result = [
+        client.compute(
```

Comment:
Why did we switch to `client.compute` here from `client.submit `?

Reply:
I did that based off your suggestion.  It fixed the hang between calling `uniform_neighbor_sample` and writing the parquet files.